ExoPlayerSampleApp/app/src/main/java/com/ooyala/sample/players/PreconfiguredIMAPlayerActivity.java

```diff
@@ -48,12 +48,12 @@ void completePlayerSetup(boolean asked) {
       playerLayoutController.addObserver(this);
       player.addObserver(this);
 
-      @SuppressWarnings("unused")
-      OoyalaIMAManager imaManager = new OoyalaIMAManager(player, skinLayout);
-
       if (player.setEmbedCode(embedCode)) {
         //player.play();
       }
+
+      @SuppressWarnings("unused")
+      OoyalaIMAManager imaManager = new OoyalaIMAManager(player, skinLayout);
```

Comment:
Shouldn't this come before the setEmbedCode statement like in two activites above this file?